test/index.test.js

```diff
@@ -619,13 +622,16 @@ rm -f docker-compose.yml
                     composeYml
                 });
 
+                const providedNodeLabel = 'foo-label';
+                const parsedAnnotations = { nodeLabel: providedNodeLabel };
+
                 jobXml = tinytim.render(TEST_JOB_XML, {
-                    nodeLabel,
```

Comment:
It would be better to add new testcase for nodelabel annotation so that we can ensure the current default `screwdriver` label works properly.

Reply:
Thx for the review ! I Added new testcase for nodeLabel annotation 👍